contracts/src/v0.8/functions/dev/v1_X/FunctionsBilling.sol

```diff
@@ -93,13 +77,13 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
 
   /// @notice Gets the Chainlink Coordinator's billing configuration
   /// @return config
-  function getConfig() external view returns (Config memory) {
+  function getConfig() external view returns (FunctionsBillingConfig memory) {
     return s_config;
   }
 
   /// @notice Sets the Chainlink Coordinator's billing configuration
-  /// @param config - See the contents of the Config struct in IFunctionsBilling.Config for more information
-  function updateConfig(Config memory config) public {
+  /// @param config - See the contents of the Config struct in FunctionsBillingConfig for more information
```

Comment:
This should say "See the contents of the FunctionsBillingConfig struct in IFunctionsBilling.sol ..."